FileSystem/WorkbookTests.js

```diff
@@ -49,13 +49,6 @@ function workbook_run_all_tests() {
         Workbook.openWorkbook(fileName1);
       });
 
-  QUnit.test(
-      'single call to workbook.open() when file is not present', function() {
-        expect(0);
-        var fileName1 = 'FileNotExist.xlsx';
-        Workbook.openWorkbook(fileName1);
-      });
-
```

Comment:
This test is to be replaced with new tests when the complete FileMapper is used.